lib/test/unit/notification.rb

```diff
@@ -65,15 +65,17 @@ def included(base)
       # Notify some information.
       #
       # Example:
-      #   def test_notification
-      #     notify("I'm here!")
-      #     # Reached here
-      #     notify("Special!") if special_case?
-      #     # Reached here too
-      #   end
+      #
+      #     def test_notification
+      #       notify("I'm here!")
+      #       # Reached here
+      #       notify("Special!") if special_case?
+      #       # Reached here too
+      #     end
```

Comment:
We can use `@example` tag for YARD: https://www.rubydoc.info/gems/yard/file/docs/Tags.md#example
But this is out-of-scope of this pull request. So we can this as is in this pull request.